store.py

```diff
@@ -88,15 +88,16 @@ def add_todo(self, description):
     def get_todos(self, task_id=None):
         query = self.session.query(TaskTable)
         if task_id is not None:
-            query = query.filter(TaskTable.id == task_id)
-        return query.all()
+            return query.get(task_id)
```

Comment:
this was actually a bug / wrong test, getting one todo should return a task object, not a list
also better to use get